Bottom: 21ca1f7828e27f79b5dca4735010e4b887f8ca48
Top:    bfacb53ab95b65e54e90656b8681f35b9bb47207
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-31 04:21:00 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!








---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 4274dd0..d510f5b 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -312,13 +312,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
